packages/patternfly-4/react-core/src/components/Popover/Popover.tsx

```diff
@@ -24,84 +27,84 @@ export const PopoverPosition = {
   right: 'right'
 };
 
-const propTypes = {
+export interface PopoverProps {
   /** Popover position */
-  position: PropTypes.oneOf(Object.keys(PopoverPosition).map(key => PopoverPosition[key])),
+  position?: BasicPlacement;
   /** If true, tries to keep the popover in view by flipping it if necessary */
-  enableFlip: PropTypes.bool,
+  enableFlip?: boolean;
   /** Popover additional class */
-  className: PropTypes.string,
+  className?: string;
   /** The reference element to which the popover is relatively placed to */
-  children: PropTypes.element.isRequired,
+  children: ReactElement<any>;
   /** Accessible label, required when header is not present */
-  'aria-label': props => {
-    if (!props.headerContent && !props['aria-label']) {
-      return new Error('aria-label is required when header is not used');
-    }
-    return null;
-  },
+  'aria-label'?: string;
   /** Header content, leave empty for no header */
-  headerContent: PropTypes.node,
+  headerContent?: ReactElement<any>;
   /** Body content */
-  bodyContent: PropTypes.node.isRequired,
-  footerContent: PropTypes.node,
+  bodyContent: ReactElement<any>;
+  footerContent?: ReactElement<any> | string;
```

Comment:
Change to `React.ReactNode` and ditch `| string` since that's included in `React.ReactNode`

Reply:
fixed